playbooks/appsemblerPlaybooks/ficus_enterprise.yml

```diff
@@ -96,3 +96,21 @@
     - { role: "xqueue", update_users: True }
     - role: "githubsshkeys"
       when: "cloud_provider == 'azure'"
+
+- name: Install Scampersand Reporting
+  hosts: "edxapp-server-0"
```

Comment:
Same as my comment above on rolling these two hosts as a list/group. If it can be done, would reduce duplication